bench/wtperf/wtperf.c

```diff
@@ -1858,7 +1858,7 @@ create_uris(CONFIG *cfg)
 	base_uri_len = strlen(cfg->base_uri);
 	cfg->uris = dcalloc(cfg->table_count, sizeof(char *));
 	for (i = 0; i < cfg->table_count; i++) {
-		uri = cfg->uris[i] = dcalloc(base_uri_len + 5, 1);
+		uri = cfg->uris[i] = dmalloc(base_uri_len + 5);
 		/*
```

Comment:
@keithbostic, this change to `dealloc` does not look safe. I think the memcpy code following depends on a null byte at the end of uri.